README.md

```diff
@@ -66,6 +66,7 @@ The `dev/cli.sh` script runs the Plane CLI, configured to connect to the local P
 
 ```bash
 dev/cli.sh connect \
+    --wait \
```

Comment:
What's this do?

Reply:
After issuing the initial request, it opens an event stream, and prints events. The command only exits when the backend is ready.